docs/how-to-guides/develop-a-tool/how-to-use-enabled-by-value.md

```diff
@@ -0,0 +1,93 @@
+# Implementing Cascading Settings for Tool Inputs
+
+Cascading settings between inputs are frequently used in situations where the selection in one input field determines what subsequent inputs should be shown.
+This approach help in creating a more efficient, user-friendly, and error-free input process.
+This article will guide you through the process of implementing cascading settings for tool inputs.
+
+## Prerequisites
+* Please ensure that your Prompt flow for VS Code is updated to version 1.2.0 or later.
+* It's crucial for you to understand the process of creating your own tool package. For thorough insights and instructions, please refer to [Create and Use Tool Package](create-and-use-tool-package.md).
```

Comment:
Not necessary to mention this in prerequisites.

Reply:
deleted